pages/[mentorUsername].page.js

```diff
@@ -120,264 +125,188 @@ function Index({ mentorDetail }) {
-                ></i>
+                  className="tw-flex tw-flex-row"
+                >
+                  <FaShareAlt className="tw-text-[28px] tw-text-[#4338CA] tw-cursor-pointer hover:tw-text-[#4338CA]/75 tw-transitional-all active:tw-scale-75"/>
+                </div>
               </div>
-              <h1>{mentorDetail.name}</h1>
-              <h3>
-                {mentorDetail.internAt} | {mentorDetail.currentStatus}
-              </h3>
-              <ul
-                className="contactLinks"
-                style={{
-                  flexDirection: "column",
-                  alignItems: "flex-start",
-                  margin: "20px 0",
-                }}
-              >
-                <li
-                  style={{
-                    display: "flex",
-                    alignItems: "center",
-                    gap: "10px",
-                    fontWeight: "600",
-                  }}
+            </div>
+            {/* About Section */}
+            <div className="tw-flex tw-flex-col">
+              {/* About Section Heading */}
+              <h1 className="tw-text-[36px] tw-font-normal tw-text-[#4338CA] tw-mb-[12px]">About</h1>
+              <p className="tw-text-justify">{mentorDetail.description}</p>
+            </div>
+          </div>
+        </div>
+        {/* Session Cards Container */}
+        <div className="tw-flex tw-flex-col tw-items-stretch tw-max-w-[448px]">
+          {/* Session Cards for every session */}
+          {mentorDetail.bookSession.length != 0 && mentorDetail.bookSession.map((session, index) => (
+            <>
+              {/* session card */}
+              <div className="tw-flex tw-flex-row tw-w-full tw-p-8 tw-bg-white tw-rounded-[36px] tw-shadow-[0px_0px_40px_0px_#00000025] tw-gap-5 tw-border-t-[1px] tw-mb-[30px]">
+                {/* Session card Icon */}
+                { session.sessionType === "video-meeting" ? <MdVideoCameraFront className="tw-text-[36px] tw-text-[#4338CA]"/>: <MdRingVolume className="tw-text-[36px] tw-text-[#4338CA]"/>}
+                <div className="tw-flex tw-flex-col">
+                  {/* Session card title */}
+                  <h1 className="tw-text-[36px] tw-font-semibold tw-text-[#4338CA]">{session.sessionName}</h1>
+                  {/* Session card description */}
+                  <h3 className="tw-text-[16px] tw-font-semibold tw-text-[#4338CA]/80">{session.sessionDescription}</h3>
+                  {/* Session card duration */}
+                  <p className="tw-flex tw-flex-row tw-items-center tw-gap-3 tw-text-[16px] tw-whitespace-nowrap tw-font-medium tw-text-black/50"><BsClock /> {session.sessionMeetingDuration} mins</p>
+                  {/* Session card Charge */}
+                  <p className="tw-flex tw-flex-row tw-items-center tw-gap-3 tw-text-[16px] tw-whitespace-nowrap tw-font-medium tw-text-black/50"><BsCurrencyRupee /> {session.priceSession} mins</p>
+                  {/* Session Book Button */}
```

Comment:
Can I have this as a component? Actually using a modular structure is very benificial. We will then pass on data. into these card

Reply:
https://github.com/anmode/grabtern-frontend/pull/331/commits/2ffc694eb320cbc9a7467a1cdc18572419893d7d
@anmode Sir,
Done 🚀🚀👍👍